bin/near-cli.js

```diff
@@ -122,6 +122,12 @@ const deploy = {
         })
         .alias({
             'accountId': ['account_id', 'contractName', 'contract_name'],
+        })
+        .option('force', {
```

Comment:
Most likely this flag will be used to do `force` actions on other commands too (`delete-key` for example). I would make it more general.